### PR TITLE
Enhancement: Switch to GitHub Actions

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] switches from Travis CI to GitHub Actions

💁‍♂️ Inspired by workflows defined in [`ergebnis/php-library-template`](https://github.com/ergebnis/php-library-template).

### CODE REVIEW

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+
+        dependencies:
+          - "locked"
```
Comment (localheinz) [inline .github/workflows/integrate.yaml:23]: Using a matrix here allows for adding additional scenarios:

- `lowest`
- `highest`

💁‍♂️ It's already accounted for in the steps below.

File: .github/workflows/integrate.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+jobs:
+  tests:
+    name: "Tests"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```
Comment (localheinz) [inline .github/workflows/integrate.yaml:20]: Using a matrix here allows for adding additional PHP versions.

**Comment by hanspagel** [comment]
Oh, that was on my list! Thank you for the PR!

**Comment by localheinz** [comment]
Thank you, @hanspagel!

**Pull request merged**